spec/directions/bake_chapter_summary_spec.rb

```diff
@@ -0,0 +1,82 @@
+require 'spec_helper'
+
+RSpec.describe Kitchen::Directions::BakeChapterSummary do
+
+  let(:chapter) do
+    chapter_element(
+      <<~HTML
+        <div data-type='page' id ="01">
+          <h1 data-type="document-title" itemprop="name">First Title</h1>
+          <section class="summary" data-element-type="section-summary">
+              <h3 data-type='title'>Summary</h3>
+              <p>Many paragraphs provide a good summary.</p>
+          </section>
+        </div>
+        <div data-type='page' id="02">
+          <h1 data-type="document-title" itemprop="name">Second Title</h1>
+          <section class="summary" data-element-type="section-summary">
+              <h3 data-type='title'>Summary</h3>
+              <p>Ooh, it's another bit of text.</p>
+          </section>
+        </div>
+      HTML
+    )
+  end
+
+  #TODO: check that it:
+  #Replaces title with h3 title (children: os-number, os-divider, os-text)
+  #
+
+  context 'when v1 is called on a chapter' do
+    it 'works' do
+      described_class.v1(chapter: chapter, metadata_source: metadata_element)
```

Comment:
Modify `metadata_element` to include an extra child that does not get copied along with the others, to test that the `metadata_source.search(...)` line only gets the parts we want.  I asked Karina to do this in another PR, see https://github.com/openstax/kitchen/pull/93/files#diff-d3c2c4afb9a47770b8067894bffac4b7e53477ae34b3e37782644c522616c04cR30